src/views/authViewHandler/index.js

```diff
@@ -55,7 +56,7 @@ class AuthViewHandler extends React.Component<Props, State> {
       return removeItemFromStorage('spectrum');
     }
 
-    if (prev.data.user !== user && user !== null) {
+    if (!_.isEqual(prev.data.user, user) && user !== null) {
```

Comment:
```suggestion
    if (user !== null && (prev.data.user === null || prev.data.user.id !== user.id)) {
```

or something like that?

Reply:
Would prefer something like this over shipping lodash :)

Reply:
@mxstbr prev.data.user is null on load so that won't work...was also worried if we're just checking equality on the id, we might miss if something else in the userObj changed (although I don't believe you're utilizing anything in the localStorage)...

thoughts on...

```    if (
      JSON.stringify(prev.data.user) !== JSON.stringify(user) &&
      user !== null
    ) {```

Reply:
There is potential in this breaking because the `user` object has a `lastSeen` field which we use to determine whether a person should receive notifications. This could change in between renders, in theory ... not totally sure, but stands out as a field that might change frequently :)

Reply:
@brianlovin just checked and lastSeen isn't in this object... just `isOnline`, which of course is useful.

Reply:
Whoops! My mistake then - that might just be a model field :P